eternalcore-core/src/main/java/com/eternalcode/core/configuration/implementation/PluginConfiguration.java

```diff
@@ -141,17 +144,96 @@ public static class RandomTeleport implements RandomTeleportSettings, DelaySetti
             "# If you want to use a static radius, set the type to STATIC_RADIUS and set the radius here.",
             "# If you using WORLD_BORDER_RADIUS, this value will be ignored."
         })
-        public int randomTeleportRadius = 1000;
+        public RandomTeleportRadiusRepresenterImpl randomTeleportStaticRadius =
+            new RandomTeleportRadiusRepresenterImpl(5000, 5000, 5000, 5000);
+        // For compatibility reasons, it must be named differently than "randomTeleportRadius".
+        // Due to limitations in the configuration library, changing the type of an existing field prevents the plugin from enabling.
 
         @Description("# Teleport to a specific world, if left empty it will teleport to the player's current world")
         public String randomTeleportWorld = "world";
 
         @Description("# Number of attempts to teleport to a random location")
         public int randomTeleportAttempts = 10;
 
+        @Description("# Unsafe blocks for random teleportation")
```

Comment:
Maybe move all the settings to separate file, list of airblocks and unsafe blocks, takes a lot of space in this already huge class

Reply:
No

Reply:
? ok

Reply:
the minecraft community is used to one config.yml file, I also work better when I don't have each separate config section separated into a separate file

Reply:
in my firm opinion, this is very unnecessary

Reply:
I was thinking about moving config settings to another class like `RtpConfiguration` and then creating an instance of this class inside the current config to separate the settings (code only). Yes, it might break older config settings connected to rtp, but it would look cleaner from the backend side of our plugin. It's only a suggestion, so we should not be so harsh about it.

Reply:
please be more precise next time. You said “file” java classes are not standard files, additionally it was not a “suggestion” because you used Requested Changes